Source/Blazorise/wwwroot/video.js

```diff
@@ -0,0 +1,66 @@
+﻿import "./vendors/ShakaPlayer";
+import * as utilities from "./utilities.js";
+
+const _instancesInfos = [];
+
+export function initialize(dotnetAdapter, element, elementId, options) {
+    element = utilities.getRequiredElement(element, elementId);
+
+    if (!element)
+        return;
+
+    // Install built-in polyfills to patch browser incompatibilities.
+    shaka.polyfill.installAll();
+
+    // Check to see if the browser supports the basic APIs Shaka needs.
+    if (shaka.Player.isBrowserSupported()) {
+        // Everything looks good!
+        initPlayer();
+    } else {
+        // This browser does not have the minimum set of APIs we need.
+        console.error('Browser not supported!');
```

Comment:
Mentioned that it is for the video component.

Reply:
I lean towards the first and will update as such. But if you don´t like it I also considered these here.

- "Browser not supported for Video player"
- "Browser not supported for Blazorise Video"
- "Browser not supported for Shaka video player"

Reply:
Yes, the first version is OK.